calblueprint/phs
# nothing is rendering - onChange onClick

[# ✨ New in this PR ✨
CRUD logic for supbase queries implemented. Created a folder for all the queries should be under the queries folder and can import queries when need to make calls to supabase
One liner: what did you do? 🛠
Created a folder for all the queries should be under the queries folder and can import queries when need to make calls to supabase.

Coverage 🙆‍♀️
How can the reviewer test your code? 👩‍💻
Go to dummy page and run it! it should automatically fetch all the displays and there are buttons to update the table and delete. Open up supbase side by side to make sure changes are being reflected

Any bugs you encountered or still having trouble with? 🐛
I am not able to test my own code - I've tried deleting the repo and redownloading everything but its still not working - andrei has tested this funtionality by running it on his laptop

Resources 📔
chat gpt

🏚 cc: @andreisito




🏚 cc: @andreisito
